cmd/template/cluster/provider/templates/azure/azure_machine_template.yaml.tmpl

```diff
@@ -11,6 +11,11 @@ metadata:
   namespace: {{ .Namespace }}
 spec:
   template:
+    metadata:
+      labels:
+        "release.giantswarm.io/version": "{{ .Version }}"
+        "giantswarm.io/organization": "{{ .Organization }}"
+        "cluster.x-k8s.io/watch-filter": "capi"
```

Comment:
This is required or the `AzureMachine` created won't be reconciled by our capi controllers using the watch filter label.